File: src/cs/mysql_block.cs
using System;
using System.Globalization;
using MySqlConnector;

namespace Calc
{
  class Program
  {
    static void Main(string[] Args)
    {
      MySqlConnection conn;
      MySqlCommand cmd;
      string query = "";
      int i;
      double x, prob;
      CultureInfo.DefaultThreadCurrentCulture = new CultureInfo("en-US");
      try
      {
        conn = new MySqlConnection("server=localhost;userid=root;password=root123;database=gaussian");
        conn.Open();
        cmd = conn.CreateCommand();
        cmd.CommandText = "DROP TABLE IF EXISTS tb_cs_block; CREATE TABLE tb_cs_block(id INTEGER AUTO_INCREMENT " +
        "PRIMARY KEY, z_score REAL NOT NULL, cumulative_distribution REAL NOT NULL)";
        cmd.ExecuteNonQuery();
        for (i = -500; i <= 500; i++)
        {
          x = (double)i / 100.0;
          prob = Gaussian.GaussianCDF(0, 1, x);
          query += $"INSERT INTO tb_cs_block(z_score, cumulative_distribution) VALUES ({x:0.00}, {prob:0.000000});";
        }
        cmd.CommandText = query;
        cmd.ExecuteNonQuery();
        conn.Close();
      }
      catch (System.Exception e)
      {
        Console.WriteLine(e.Message);
      }
    }
  }
}

File: src/c/mysql_block.c
#include <stdio.h>
#include <stdlib.h>
#include <mysql.h>
#include "gaussian.h"

int main(int argc, char **argv)
{
  MYSQL *conn = mysql_init(NULL);
  int exitCode, i, buffer = 0;
  double x, prob;
  char *query = (char *)malloc(90000 * sizeof(*query));
  if (mysql_real_connect(conn, "localhost", "root", "root123", "gaussian", 3306, NULL, CLIENT_MULTI_STATEMENTS) == NULL)
  {
    fprintf(stderr, "Connection to database failed: %s\n", mysql_error(conn));
    exit(-1);
  }
  exitCode = mysql_query(conn, "DROP TABLE IF EXISTS c_block; CREATE TABLE c_block(id INTEGER AUTO_INCREMENT "
                               "PRIMARY KEY, z_score REAL NOT NULL, cumulative_distribution REAL NOT NULL)");
  if (exitCode)
  {
    fprintf(stderr, "%s\n", mysql_error(conn));
    exit(-1);
  }
  mysql_next_result(conn);
  for (i = -500; i <= 500; i++)
  {
    x = (double)i / 100.0;
    prob = gaussianCDF(0, 1, x);
    buffer += sprintf((char *)(query + buffer), "INSERT INTO c_block(z_score, cumulative_distribution) VALUES (%.2f, %f);", x, prob);
  }
  exitCode = mysql_query(conn, query);
  if (exitCode)
  {
    fprintf(stderr, "%s\n", mysql_error(conn));
    exit(-1);
  }
  mysql_close(conn);
  return 0;
}